pdns/recursordist/docs/changelog/4.4.rst

```diff
@@ -0,0 +1,241 @@
+Changelogs for 4.4.x
+====================
+
+.. changelog::
+  :version: 4.4.0-alpha1
+  :released: 22th of April 2020
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 9031
+    :tickets: 9025
+
+    Fix compilation of the ports event multiplexer.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9000
+
+    Fix warnings with llvm10 and -Wrange-loop-construct (Kirill Ponomarev).
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8985
+
+    Fix compilation without deprecated OpenSSL APIs (Rosen Penev).
+
+  .. change::
+    :tags: New Features
+    :pullreq: 8967
+
+    Implement native DNS64 support, without Lua.
+
+  .. change::
+    :tags: New Features
+    :pullreq: 8927
+
+    Add custom tags to RPZ hits.
+
+  .. change::
+    :tags: New Features
+    :pullreq:  8910
+
+    Allow attaching a 'routing' tag string to a query in lua code and use that
+    tag in the record cache when appropriate.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8900
+    :tickets: 8739
+
+    Detect {Libre,Open}SSL functions availability during configure.
+
+  .. change::
+    :tags: New Features
+    :pullreq: 8898
+
+    Share record cache between threads.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8887
+
+    Better handling of reconnections in Remote Logger.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8883
+    :tickets: 8629
+
+    Add 'queue full' metrics for our remote logger, log at debug only.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8876
+    :tickets: 8875
+
+    Update boost.m4
+
+  .. change::
+    :tags: New Features
+    :pullreq: 8874
+
+    Add support for Proxy Protocol between dnsdist and the recursor.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8864
+    :tickets: 8861
+
+    Only log qname parsing errors when 'log-common-errors' is set.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8812
+
+    Keep a masked network in the Netmask class.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8631
+
+    Replace include guard ifdef/define with pragma once (Chris Hofstaedtler).
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 8826
+    :tickets: 8825
+
+    Refuse NSEC records with a bitmap length > 32.
```

Comment:
This was backported to 4.3.x so perhaps we should not mention it there?

Reply:
IMO even if backported I think it is good to mention.

Reply:
I would really expect the ChangeLog for 4.4.0 to list the changes between the latest 4.3.x release and 4.4.0, otherwise what does it represent exactly?

Reply:
I'm using the series of commits from the 4.3.x branch point till now.

Reply:
Right, so that represents the list of changes if you upgrade from 4.3.0-alpha1 to 4.4.0?

Reply:
Yes.  In my head I use the model of following master. But if you look at it like you do it does really make sense to skip the backported ones.